web/mocksite/index.pug

```diff
@@ -72,3 +72,14 @@ block main
 	section
 		h2 subscription plans
 		xiome-subscription-planner
+
+	section
+	xio-code
+		h2 xio-code copiable block
```

Comment:
- indentation here isn't placing the xio-code stuff underneath the section; it should be
- we don't want to have an h2 underneath the xio-code element, the h2 is just a header to denote the section